bite-size-notes/week-8-d-Probability and Recursive Bayesian Filtermd.md

```diff
@@ -0,0 +1,210 @@
+### Relationship
+The relationship between probability and the recursive Bayesian filter is fundamentally rooted in Bayes theorem, which provides a mechanism for updating the probability of a hypothesis based on new evidence. Bayes theorem states that the posterior probability, the updated probability after considering new data, is proportional to the product of the prior probability and the likelihood of the observed data. This iterative updating process is the core of the recursive Bayesian filter.
+
+The Recursive Bayesian Filter operates by applying Bayes theorem in a recursive manner. Initially, a prior distribution represents the initial belief about the state or parameters of the system. As new data points are observed, the prior distribution is updated to a posterior distribution using the likelihood of the new data. This posterior distribution then becomes the new prior for the next iteration. This recursive process allows for continuous refinement of the estimates, making the approach highly adaptive and responsive to new information.
+## Preliminaries
+First, we'll review the variables and space of probability and what each of them means.
+### Random variables
+$\mathbf{X}$: A quantity whose value is unknown/uncertain to us.
+
+$\mathbf{x}$: A possible value of $\mathbf{X}$, or sample of $\mathbf{X}$, or realization of $\mathbf{X}$.
+### Probability space (Ω，F，P)
+Ω: Sample space ( all possible outcomes of random variable ).
+
+F: Event space( all possible sets of outcomes in Ω ).
+
+P: Probability measure (assigns probability to each event ).
+### Basis of probability
+And back to the function：
+
+The characteristic of continuous space is that random variables can take continuous values. Assume that all continuous random variables in this text have a probability density function (PDF). The common density function is the one-dimensional normal distribution (normal distribution) with mean $\mu$ and variance $\sigma^2$. The probability density function of the normal distribution can be given by the Gaussian function:
+
+$$
+p(x) = (2\pi\sigma^2)^{-\frac{1}{2}} \exp \left( -\frac{1}{2} \frac{(x - \mu)^2}{\sigma^2} \right) \tag{1} 
+$$
+
+This can be abbreviated as $N(x; \mu, \sigma^2)$.
+
+Equation (1) assumes that $x$ is a scalar quantity, but $x$ is often a multi-dimensional vector. The normal distribution of vectors is called multivariate. The density function of the multivariate normal distribution is:
+
+$$
+p(x) = \det(2\pi\Sigma)^{-\frac{1}{2}} \exp \left( -\frac{1}{2} (x - \mu)^T \Sigma^{-1} (x - \mu) \right) \tag{2}
+$$
+
+
+where $\mu$ is the mean vector, $\Sigma$ is a positive semidefinite and symmetric matrix called the covariance matrix; $T$ denotes the transpose of the vector.
+
+The joint distribution of two random variables $\mathbf{X}$ and $\mathbf{Y}$ is:
```

Comment:
Should give a brief description of what a joint distribution is

Reply:
Thank you, I have added a brief explanation.